### PR TITLE
Add new ui test to quay in prow CI - v2

### PR DESCRIPTION
**Pull request by huweihua-redhat**
What does this PR do:

   *  Add quay new UI test

### CODE REVIEW

**Comment by huweihua-redhat** [comment]
/pj-rehearse  periodic-ci-quay-quay-tests-new_ui_add_tag-quay-newui-quay-e2e-tests-quay311-ocp416-newui

File: ci-operator/config/quay/quay-tests/quay-quay-tests-new_ui_add_tag__quay-newui.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+base_images:
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
```
Comment (kuiwang02) [inline ci-operator/config/quay/quay-tests/quay-quay-tests-new_ui_add_tag__quay-newui.yaml:7]:
```yaml
  image_stream_tag:
    name: release
    namespace: openshift
    tag: golang-1.21
```
to
```yaml
  image_stream_tag:
    name: builder
    namespace: ocp
    tag: rhel-8-golang-1.21-openshift-4.16
```

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-new_ui_add_tag-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-new_ui_add_tag-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-new_ui_add_tag-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Pull request closed without merging**